source/index.js

```diff
@@ -0,0 +1,109 @@
+"use strict";
+
+var fs = require( "fs" );
+var parseString = require( "xml2js" ).parseString;
+
+var parse = {};
+
+var unpackage = function ( report )
+{
+    var packages = report.package;
+
+    var output = []
+
+    packages.forEach( function ( pack )
+    {
+        var cov = pack.sourcefile.map( function (s) 
+        {
+            var fullPath = pack.$.name + '/' + s.$.name
+            var className = fullPath.substring(0, fullPath.lastIndexOf('.'));
+
+            var c;
+            pack.class.forEach( function( cl ) {
+                if (cl.$.name == className) {
+                    c = cl;
+                }
+            });
```

Comment:
Using `forEach` to search for a matching value like this is a little odd. Maybe use `filter` to find a match and then use the first result instead?